src/app/contact/contact.component.html

```diff
@@ -0,0 +1,174 @@
+<div class="container">
```

Comment:
All styling should be done in CSS/SCSS files.

Reply:
@strizzwald fixed